Storage/SQL/SQLiteHistory.swift

```diff
@@ -8,6 +8,8 @@ import XCGLogger
 
 private let log = Logger.syncLogger
 
+private let TopSitesCacheSize = 100
```

Comment:
Naaaaah. There are a few ways to do this:

* Compute the max number of cells, just as we do in TopSitesPanel.
* Get told the max the first time we need to use it, and assume that it'll never change.
* Make it part of the cache pref, and if it gets larger, invalidate the cache and use the larger value.
* Figure out the largest for any device we support, and put that here. (The laziest option!)

We shouldn't use 100 here, because we'll be needlessly computing 100 top sites when we'll never show more than, say, 16.